service/controller/v12/resource/instance/azure.go

```diff
@@ -0,0 +1,180 @@
+package instance
+
+import (
+	"context"
+	"fmt"
+	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2018-06-01/compute"
+	"github.com/Azure/go-autorest/autorest/to"
+	providerv1alpha1 "github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
+	"github.com/giantswarm/azure-operator/service/controller/v12/key"
+	"github.com/giantswarm/microerror"
+)
+
+func (r *Resource) reimageInstance(ctx context.Context, customObject providerv1alpha1.AzureConfig, instance *compute.VirtualMachineScaleSetVM, deploymentNameFunc func(customObject providerv1alpha1.AzureConfig) string, instanceNameFunc func(customObject providerv1alpha1.AzureConfig, instanceID string) (string, error)) error {
```

Comment:
Did this function just move to a different file? What changed? It is hard to tell for me with such a big PR / confusing diff.

Reply:
The refactoring is reverted now, closing this conversation